velox/connectors/hive/storage_adapters/abfs/tests/AbfsFileSystemTest.cpp

```diff
@@ -15,29 +15,86 @@
  */
 
 #include "velox/connectors/hive/storage_adapters/abfs/AbfsFileSystem.h"
+#include <azure/storage/files/datalake.hpp>
 #include "gtest/gtest.h"
 #include "velox/common/base/tests/GTestUtils.h"
 #include "velox/common/file/File.h"
 #include "velox/common/file/FileSystems.h"
 #include "velox/connectors/hive/FileHandle.h"
 #include "velox/connectors/hive/HiveConfig.h"
 #include "velox/connectors/hive/storage_adapters/abfs/AbfsReadFile.h"
+#include "velox/connectors/hive/storage_adapters/abfs/AbfsWriteFile.h"
 #include "velox/connectors/hive/storage_adapters/abfs/tests/AzuriteServer.h"
 #include "velox/exec/tests/utils/PortUtil.h"
 #include "velox/exec/tests/utils/TempFilePath.h"
 
 #include <atomic>
+#include <filesystem>
 #include <random>
 
 using namespace facebook::velox;
-
+using namespace facebook::velox::filesystems::abfs;
+using namespace Azure::Storage::Files::DataLake;
 using ::facebook::velox::common::Region;
 
 constexpr int kOneMB = 1 << 20;
 static const std::string filePath = "test_file.txt";
 static const std::string fullFilePath =
     facebook::velox::filesystems::test::AzuriteABFSEndpoint + filePath;
 
+// A mocked blob storage file client backend with local file store.
+class MockBlobStorageFileClient : public IBlobStorageFileClient {
```

Comment:
Consider moving this to another file to simplify.

Reply:
Consider adding `AbfsFileSystemWriteTest.cpp` and renaming the existing test file to `AbfsFileSystemReadTest.cpp` if feasible.

Reply:
> Consider adding `AbfsFileSystemWriteTest.cpp` and renaming the existing test file to `AbfsFileSystemReadTest.cpp` if feasible.

hey @majetideepak the test case not only have `read` and `write` part, also inlcuding other ops as `cred` check, `list/rename/mkdir` etc , so just split to `ReadTest` and `WriteTest` looks not a good refactor.

I would like to keep still one test file in this PR, and send antoher seperate PR do refactor if needed.

Reply:
Can you add the `MockBlobStorageFileClient` introduced here to a separate .h/.cpp

Reply:
sure @majetideepak missed that part before